salt/annotations/config/etc-nginx-sites-enabled-annotations.conf

```diff
@@ -13,13 +13,7 @@ server {
         include fastcgi_params;
         fastcgi_intercept_errors on;
         fastcgi_param SCRIPT_FILENAME $request_filename;
-        {% if pillar.annotations.containerized %}
         fastcgi_pass localhost:9000;
-        {% else %}
```

Comment:
don't need this anymore, as the host-wide fpm is used only by dummies rather than by `annotations`

Reply:
(was a feature flag)